features/e2e_tests/cpu-manager_test.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+function local_env() {
+    BASEDIR="$(dirname "$0")"
+    source "${BASEDIR}/../lib/functions.sh"
+}
+
+
+function validate_function_by_worker() {
+    WORKERS=("$(oc get node -l node-role.kubernetes.io/worker -o jsonpath="{range .items[*]}{.metadata.name} ")")
+    if [ -z "${WORKERS}" ];then
+        echo "===> No workers detected, exiting..."
```

Comment:
You can use the `lib/functions.sh` functions that has some `info`, `err`, `die` handy functions.